### PR TITLE
Test changes

### PR DESCRIPTION
**Pull request by Lesenr1**

### CODE REVIEW

File: test/multisig.test.jsligo [hunk 1]
```diff
@@ -89,9 +83,8 @@ const create_proposal_params: Main.Parameter.Types.proposal_params = {
     transfers: transfers
 };
 
-export const _test_signer_creates_proposal_works = (mainf: func_main): unit => {
-    const [taddr, _1, _2] = Test.originate(mainf, storage, 0 as tez);
-    const contr = Test.to_contract(taddr);
+export const test_signer_creates_proposal_works = do {
```
Comment (SuzanneSoy) [inline test/multisig.test.jsligo]:
```suggestion
const test_signer_creates_proposal_works = do {
```

No need to export a `const … : unit`

File: test/multisig.test.jsligo [hunk 2]
```diff
@@ -122,22 +115,21 @@ export const _test_signer_creates_proposal_works = (mainf: func_main): unit => {
...
             0 as tez
         );
     assert_error(fail_tx, Main.Conditions.Errors.only_signer);
-    const curr_storage = Test.get_storage(taddr);
+    const curr_storage = Test.get_storage(orig.addr);
     assert(curr_storage == storage);
 };
 
-export const _test_signer_signs_proposal_works = (mainf: func_main): unit => {
+export const test_signer_signs_proposal_works = do {
```
Comment (SuzanneSoy) [inline test/multisig.test.jsligo]:
```suggestion
const test_signer_signs_proposal_works = do {
```

File: test/multisig.test.jsligo [hunk 3]
```diff
@@ -176,55 +167,48 @@ export const _test_signer_signs_proposal_works = (mainf: func_main): unit => {
...
     assert_error(fail_tx, Main.Conditions.Errors.only_signer);
-    const curr_storage = Test.get_storage(taddr);
+    const curr_storage = Test.get_storage(orig.addr);
     assert(curr_storage == storage);
 };
 
-export const _test_unknown_signs_unknown_proposal_fails = (mainf: func_main): unit => {
-    const [taddr, _1, _2] = Test.originate(mainf, storage, 0 as tez);
-    const contr = Test.to_contract(taddr);
+export const test_unknown_signs_unknown_proposal_fails = do {
```
Comment (SuzanneSoy) [inline test/multisig.test.jsligo:184]:
```suggestion
const test_unknown_signs_unknown_proposal_fails = do {
```

File: test/multisig.test.jsligo [hunk 4]
```diff
@@ -176,55 +167,48 @@ export const _test_signer_signs_proposal_works = (mainf: func_main): unit => {
...
+            (Sign_proposal(2 as nat) as parameter_of Main),
             0 as tez
         );
     assert_error(fail_sign_tx, Main.Conditions.Errors.only_signer);
 };
 
-export const _test_signer_signs_unknown_proposal_fails = (mainf: func_main): unit => {
-    const [taddr, _1, _2] = Test.originate(mainf, storage, 0 as tez);
-    const contr = Test.to_contract(taddr);
+export const test_signer_signs_unknown_proposal_fails = do {
```
Comment (SuzanneSoy) [inline test/multisig.test.jsligo]:
```suggestion
const test_signer_signs_unknown_proposal_fails = do {
```

File: test/multisig.test.jsligo [hunk 5]
```diff
@@ -176,55 +167,48 @@ export const _test_signer_signs_proposal_works = (mainf: func_main): unit => {
     assert(end_signed_proposal.approved_signers == expected_signer);
 };
 
-export const _test_unknown_signs_proposal_fails = (mainf: func_main): unit => {
-    const [taddr, _1, _2] = Test.originate(mainf, storage, 0 as tez);
-    const contr = Test.to_contract(taddr);
+export const test_unknown_signs_proposal_fails = do {
```
Comment (SuzanneSoy) [inline test/multisig.test.jsligo:170]:
```suggestion
const test_unknown_signs_proposal_fails = do {
```

File: test/multisig.test.jsligo [hunk 6]
```diff
@@ -176,55 +167,48 @@ export const _test_signer_signs_proposal_works = (mainf: func_main): unit => {
...
             0 as tez
         );
     assert_error(fail_sign_tx, Main.Conditions.Errors.no_proposal_exist);
 };
 
-export const _signer_creates_proposal_signers_sign_proposal_execute_works = (
-    mainf: func_main,
-    mainfa2: func_mainfa2
-): unit => {
+export const signer_creates_proposal_signers_sign_proposal_execute_works = do {
```
Comment (SuzanneSoy) [inline test/multisig.test.jsligo]:
```suggestion
const signer_creates_proposal_signers_sign_proposal_execute_works = do {
```

File: test/multisig.test.jsligo [hunk 7]
```diff
@@ -315,15 +298,11 @@ export const _signer_creates_proposal_signers_sign_proposal_execute_works = (
     assert(owner == bob);
 };
 
-export const _signer_creates_proposal_signers_sign_proposal_execute_over_threshold_works = (
-    mainf: func_main,
-    mainfa2: func_mainfa2
-): unit => {
+export const _signer_creates_proposal_signers_sign_proposal_execute_over_threshold_works = do {
```
Comment (SuzanneSoy) [inline test/multisig.test.jsligo]:
```suggestion
const _signer_creates_proposal_signers_sign_proposal_execute_over_threshold_works = do {
```

**Review by SuzanneSoy — APPROVED** [review]
LGTM with suggestions

**Review by LigoSuzanneSoy — APPROVED** [review]

**Pull request merged**